BUILDING.md

```diff
@@ -154,6 +154,12 @@ cargo make \
 
 To use the image in Amazon EC2, we need to register the image as an AMI.
 
+To do this, you'll need to have your AWS account credentials setup on your system.
+There are lots of ways to do this but usually it's through [the `aws` CLI](https://aws.amazon.com/cli/) via it's `configure` command
+with your user's access and secret keys.
```

Comment:
Our project convention for markdown is to have one sentence per line, regardless of line length. (I'm still trying to adjust to that! :D)

Reply:
Oh gotcha! I was not aware! Thanks Sean!